intro-to-object-oriented-programming/practice-intro-to-oop.md

```diff
@@ -8,16 +8,866 @@ Complete all questions below.
 
 <!-- Two questions -->
 
+<!-- Question 1 -->
```

Comment:
Is the number auto-generated or is it manually maintained? If manual, is it really worth maintaining? If we insert additional questions do we want to have to renumber the questions? It seems like Learn will generate its own numbers regardless of the commented number, which could lead to confusion if they get out of sync.

Reply:
This isn't auto-generated, it's manually maintaned!

I agree with you, (it's a reason why I really dislike numbering the questions in the challenge title itself)

I'm cool with dropping this practice (I think I'm the only one that does it anyway LOL), but it happens to be helpful when you're writing questions and scrolling really hard and all the formatting makes your eyes glaze over.

When I've been in Learn (previewing or otherwise), when I want to find the solution to a problem (think code challenge), I've found that cmd+f'ing the string `Question X` is a little bit more consistent and direct than cmd+f'ing the question text itself. That's my explanation, but I'll prob just drop it if it's noise to other folks LOL